src/ui/App.qml

```diff
@@ -233,8 +233,17 @@ Rectangle {
                     SplitButton {
                         id: renderBtn;
                         btn.accent: true;
-                        text: isAddToQueue? (render_queue.editing_job_id > 0? qsTr("Save") : qsTr("Add to render queue")) : qsTr("Export");
-                        iconName: "video";
+                        function getMainText() {
+                            if (addQueueDelayed) {
+                                return qsTr("Added to queue");
+                            } else if (isAddToQueue) {
+                                return render_queue.editing_job_id > 0? qsTr("Save") : qsTr("Add to render queue");
+                            } else {
+                                return qsTr("Export");
+                            }
+                        }
+                        text: getMainText();

```

Comment:
this can be written as:
```qml
text: {
    if (addQueueDelayed) {
        return qsTr("Added to queue");
    } else if (isAddToQueue) {
        return render_queue.editing_job_id > 0? qsTr("Save") : qsTr("Add to render queue");
    } else {
        return qsTr("Export");
    }
};
```